src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -364,9 +371,12 @@ def shim_to_feedback(
             feedback_event["platform"] = event.platform
             feedback_event["level"] = event.data["level"]
             feedback_event["environment"] = event.get_environment().name
-            feedback_event["tags"] = [list(item) for item in event.tags]
 
-        else:
+            tags = {}
```

Comment:
convert the tags to a `dict` because it was causing issues in the shim pipeline

Reply:
We don't to force this conversion anymore after doing it in create_feedback_issue. The shim tests will pass since they check the kafka call, not the shim fx itself.